### DISCUSSION TITLE
Header in outline panel but no in the rendered document

### DISCUSSION THREAD

**Discussion by debdagybra** [quarto-dev/quarto-cli#9919] [Show and Tell]
### Description

I would like to share some piece of code I was very happy to find.

I wanted to add entries in my outline panel to structure my code but without seeing them in the rendered document.  
I found this [working code](https://stackoverflow.com/questions/76795848/quarto-how-to-hide-headings-in-html-render-but-keep-in-rstudio-document) on stackoverflow but only for the html format. 

But today, I found a way working with all formats.

Edited as suggested by mcanouil

```
---
title: "Test to hide headers"
format: typst
---

::: {.content-hidden}
# invisible title
:::

some text

# title 1

some text

```

![image](https://github.com/quarto-dev/quarto-cli/assets/10846318/3bc960f9-339a-4e2a-8df7-b91a84c4fa99)

![image](https://github.com/quarto-dev/quarto-cli/assets/10846318/c11b55e5-343b-413f-ad83-1eb7cad08def)

I hope this will be usefull to other people as well.

**Comment by mcanouil** [comment]
Thanks for sharing!

Note that the following should be enough:

````
::: {.content-hidden}
# invisible title
:::
````

**Reply by mcanouil** [reply]
In passing the `.hidden` trick showed does not have the same purpose as `.content-*` used for conditional content.

- `.hidden` hides but includes so you would see the content in some places such as table of content etc. Note that it also (as far as I know) only works for HTML-based content. It is mostly documented for equations.
  See <https://quarto.org/docs/authoring/markdown-basics.html#equations>

- `.content-hidden` does not include the content (code cells inside it are still evaluated/executed) in the resulting document.
  See <https://quarto.org/docs/authoring/conditional.html>

**Reply by debdagybra** [reply]
Thanks for the additional information and the shorter code !
